jniFaiss/src/com_amazon_opendistroforelasticsearch_knn_index_faiss_KNNFIndex.cpp

```diff
@@ -0,0 +1,272 @@
+#include "com_amazon_opendistroforelasticsearch_knn_index_faiss_KNNFIndex.h"
```

Comment:
This class has core logic and would need some insights into way faiss library operates to review the code. I plan to visit this code once I have understanding of faiss interface.